apps/ledger-live-mobile/src/newArch/features/Accounts/components/AddAccountButton/index.tsx

```diff
@@ -73,19 +57,11 @@ const AddAccountButton: FC<Props> = ({ sourceScreenName, disabled, currency, onC
 
   return (
     <>
-      <StyledPressable
+      <AddAccountButtonComponent
+        label={t("addAccounts.addNewOrExisting")}
+        onClick={handleOpenAddAccountModal}
         disabled={disabled}
-        style={({ pressed }: { pressed: boolean }) => [
-          { opacity: pressed ? 0.5 : 1.0, marginVertical: 12 },
-          disabled && { opacity: 0.5 },
-        ]}
-        hitSlop={6}
-        onPress={handleOpenAddAccountModal}
-        testID="add-new-account-button"
-      >
-        <Text variant="large">{t("addAccounts.addNewOrExisting")}</Text>
-        <Icons.Plus size="S" color="neutral.c100" />
-      </StyledPressable>
+      />
```

Comment:
```suggestion
testID="add-new-account-button"
      />
```

add the property to the component please :)